addons/mail/static/src/utils/test_utils.js

```diff
@@ -169,47 +170,99 @@ function nextAnimationFrame() {
 }
 
 //------------------------------------------------------------------------------
-// Public: test lifecycle
+// Model definitions setup
 //------------------------------------------------------------------------------
 
-function beforeEach(self) {
-    const data = MockModels.generateData();
+const TEST_USER_IDS = {
+    partnerRootId: 2,
+    currentPartnerId: 3,
+    currentUserId: 2,
+    publicPartnerId: 4,
+    publicUserId: 3,
+};
 
-    data.partnerRootId = 2;
-    data['res.partner'].records.push({
-        active: false,
-        display_name: "OdooBot",
-        id: data.partnerRootId,
-    });
+let modelDefinitions;
+QUnit.done(() => {
+    // clear model definitions only once all the tests are done
+    modelDefinitions = undefined;
+});
```

Comment:
Is it actually necessary? It probably doesn't hurt but I can't really see why we need to do it?